.github/workflows/main.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  jjb:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    name: Jenkins Job Builder
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '2.7'
+      - name: Restore pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
```

Comment:
We don't have a `requirements.txt`. I think that means the cache will never be rebuilt.

@evgeni would a solution to be to add a `pip freeze > requirements-lock.txt` at the end of the install phase?

Reply:
Technically? Yeah.

But we also could just dump the three packages on the next step into a `requirements.txt` and use that? That would produce a rather static hash, as the non-versioned deps don't change much, but I think that's still OK?